CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 - Properly handle `.freqs` in `output_monitors` of adjoint plugin.
+- Simulation updater for `.hdf5` files with custom data.
```

Comment:
I only tried custom source and custom medium and yeah for some reason custom medium seemed fine. However I haven't tried other custom data, so not sure. Also not sure if there's a way to create the custom medium in which it will error (e.g. passing an xarray object as opposed to one of our inherited ones?) So maybe safer to leave as is?

Reply:
sure sounds good